src/features/navigation/RootNavigator.tsx

```diff
@@ -30,6 +31,12 @@ import { ColorsEnum } from 'ui/theme'
 import { onNavigationStateChange } from './services'
 import { TabNavigator, TabParamList } from './TabBar/TabNavigator'
 
+/**
+ * WARNING !
+ * Deeplink: When updating the screen parameters, pay attention to the deeplink handlers.
+ * If a deeplink handler indexes the screen you're changing the route's parameters,
+ * please update the deeplink handler in consequence.
```

Comment:
Normalement le typage est sensé prévenir / bloquer le dev avant de merger. Tu peux vérifier si tu veux en modifiant ici un des objets params. 
Néanmoins, je suis ok avec le commentaire. 
Pour simplifier : 
```suggestion
 * Deeplink: When updating the screen parameters, pay attention to the deeplink handlers.
 * If a deeplink handler indexes the screen with params you are changing, please update the deeplink handler in consequence.
```